apps/site/assets/js/support-form.js

```diff
@@ -348,6 +348,13 @@ const validators = {
   },
   recaptcha: function($) {
     return !!$("#g-recaptcha-response").val();
+  },
+  vehicle: function($) {
+    const value = $("#vehicle").val();
+    if (value && value.length > 0) {
+      return /^[0-9]+$/.test(value);
+    }
+    return true;
```

Comment:
Does `/^[0-9]*$/` cover the empty case as well? Would simplify the function.

Reply:
How so? Empty cases are valid since the field is optional, so I'm just testing non-empty cases here.

Reply:
Right, so something like:
```
const value = $("#vehicle").val();
return /^[0-9]*$/.test(value)
```

Reply:
Ahh I missed the subtle `+` to `*` change there...